docs/tutorials/hello-world-application/using-the-database.mdx

```diff
@@ -531,7 +539,7 @@ Once this transaction is executed, the record will be deleted from the database.
 
 ## Read-Only Queries
 
-In addition to using actions to read data from the database, we can also use read-only queries. These queries are executed on a single node and do not require a transaction to be broadcasted to the network. This is useful for reading data from the database without incurring any gas fees.
+In addition to using `view` actions to read data from the database, we can also use write `SELECT` queries.
```

Comment:
I don't think SELECT should be backticked

Reply:
addressed here: [0d326c6](https://github.com/kwilteam/docs/pull/34/commits/0d326c6371b4dff3b82bcbcc1e41dafa1900bc24)